bin/remove_field.py

```diff
@@ -89,19 +79,22 @@ def wait_times_access():
     wait_time_generator=wait_times_access,
     module_logger=logger,
 )
-def fix_data_in_s3(fields, bucket, s3_client, crash_data):
```

Comment:
This `s3_client` is a boto client. We swap it for a `s3_crashstorage` which is a `socorro.external.boto.crashstorage.S3CrashStorage` instance which has a `connection` that's the `ConnectionContext`. This means that `remove_field` is using more of our API and less of the boto API directly.